nlp/tests.py

```diff
@@ -21,6 +21,29 @@ def test_strip_spaces(self):
         self.assertEqual(result, expected)
 
 
+class W2NTestCase(TestCase):
+    def test_default_lang(self):
+        text = "one two three"
+        result = words_to_numbers(text)
+        expected = "1 2 3"
+
+        self.assertEqual(result, expected)
+
+    def test_spanish_lang(self):
+        text = "uno dos tres"
+        result = words_to_numbers(text, language="es")
+        expected = "1 2 3"
+
+        self.assertEqual(result, expected)
+
+    def test_unsupported_lang(self):
```

Comment:
Does this work? When I tested this 2 weeks ago slovak was not supported

Reply:
I think its unrelated - we could simply create new PR to the w2n repository.